matchstart.fbs

```diff
@@ -75,9 +75,8 @@ table PlayerConfiguration {
   location:string (required);
   run_command:string (required);
   loadout:PlayerLoadout;
-  /// In the case where the requested player index is not available, spawnId will help
-  /// the framework figure out what index was actually assigned to this player instead.
   spawn_id:int;
+  group_id:string (required);
```

Comment:
You went with `group_id`. Not bad. Having googled all sorts of synonyms I think I prefer `kind_id` or `type_id` or `rlbot_id`. What do you think?

Reply:
I personally thing `kind_id` & `type_id` are too vague - `rlbot_id` could be ok, I did consider that and it makes sense, I just wanted to avoid the word `rlbot` in a variable name.

Reply:
`actor_id` and `agent_id` are next on my list. But both have issues and sound like concrete instance ids instead of ids for a bot/script kind. `persona`? Or we say, fuck it and use `bot_id` even for scripts.

Reply:
I like `agent_id` - `actor_id` is a term used for another identifier in core.

Reply:
I accept `agent_id`. I don't think we find better.